### PR TITLE
Fix typo in sv translation

### PR DESCRIPTION
**Pull request by PhPersson**
## Description

Fixed typo in SV translation

## Related Issue

This PR fixes or closes issue: fixes #

## Types of changes

-   [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
-   [ ] 🚀 New feature (non-breaking change which adds functionality)
-   [x] 🌎 Translation (addition or update a translation)
-   [ ] ⚙️ Tech (code style improvement, performance improvement or dependencies bump)
-   [ ] 📚 Documentation (fix or addition in the documentation)
-   [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

-   [ ] My code follows the code style of this project.
-   [ ] My change requires a change to the documentation.
-   [ ] I have updated the documentation accordingly.
-   [ ] I have tested the change locally.
-   [ ] I followed [the steps](https://github.com/piitaya/lovelace-mushroom#maintainer-steps-to-add-a-new-language) if I add a new language .

### CODE REVIEW

**Review by piitaya — APPROVED** [review]
.

File: src/translations/sv.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
         "incompatible_controls": "Kontroller som inte stöds av enheten kommer inte visas."
       },
       "fan": {
+        "icon_animation": "Animera ikonen när fläkten är på?"
```
Comment (piitaya) [inline src/translations/sv.json:52]:
```suggestion
        "icon_animation": "Animera ikonen när fläkten är på?",
```

**Review by piitaya — CHANGES_REQUESTED** [review]

**Review by piitaya — CHANGES_REQUESTED** [review]

**Comment by piitaya** [comment]
Thank you for the translation fixes. But please test the changes locally before opening the PR.

File: src/translations/sv.json [hunk 2]
```diff
@@ -72,6 +73,9 @@
       "weather": {
         "show_conditions": "Förhållanden?",
         "show_temperature": "Temperatur?"
+      },
+      "vacuum": {
+        "icon_animation": "Animera ikonen när dammsugaren är på?"
```
[the remark this answers is not in the document]
Comment (PhPersson) [reply src/translations/sv.json]: This should be fixed now.

**Comment by PhPersson** [comment]
Fixed your comments now. Fixing on mobile late at night is not optimal. 
Changed the transkation to a more generic one now!

**Review by piitaya — APPROVED** [review]

**Comment by piitaya** [comment]
Perfect! 👌

**Pull request merged**